ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -291,6 +292,7 @@ qualifiedName
 identifier
     : IDENTIFIER             #unquotedIdentifier
     | QUOTED_IDENTIFIER      #quotedIdentifierAlternative
+    | STRING                 #stringIdentifier
```

Comment:
ANSI standard is to have `"` for identifiers and `'` for constants - most DBs don't really follow this (Postgres does, but MySQL and SQL-Server don't). Since we use `identifier` in `tableProperties` it's either we allow `'` or we deal with needing to do `"foo"='bar'` which I think is incredibly confusing

Reply:
Why do we need to add `STRING` here?

Reply:
All of the other syntax only allow for `"` for identifiers, not `'` - I just reused `STRING` because it allows for `'`.

Reply:
Talked offline here. The problem is that identifiers and literals need to be differentiated in the select statements (e.g. `SELECT 'foo'` and `SELECT "foo"` are different, one refers to literals the other refers to a column)